src/components/footer/Footer.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
+
+function RenderFooter() {
+  return (
+    <Footer>
+      <Copywright>
+        <p>
+          © Sascha Collinet | Allaround Family - Your Family Friendly Activity
+          Finder 2021
+        </p>
+      </Copywright>
+      <Profiles>
+        <ProfilesList>
+          <li>
+            <ProfilesAnchor
+              href="https://github.com/saschacollinet"
+              target="_blank"
+            >
+              GitHub
+            </ProfilesAnchor>
+          </li>
+          <li>
+            <ProfilesAnchor
+              href="https://www.linkedin.com/in/sascha-collinet/"
+              target="_blank"
+            >
+              LinkedIn
+            </ProfilesAnchor>
+          </li>
+          <li>
+            <ProfilesAnchor
+              href="https://www.xing.com/profile/Sascha_Collinet/cv"
+              target="_blank"
+            >
+              Xing
+            </ProfilesAnchor>
+          </li>
+        </ProfilesList>
+      </Profiles>
+    </Footer>
+  )
+}
+
+const Footer = styled.div`
+  text-align: center;
+  position: absolute;
+  bottom: 20px;
+`
+
+const Copywright = styled.div`
+  margin-bottom: 20px;
+`
+
+const Profiles = styled.div``
+
+const ProfilesList = styled.ul`
+  list-style: none;
+`
+
+const ProfilesAnchor = styled.a`
+  color: black;
+  text-decoration: none;
```

Comment:
It is always better to give the link elements some sort of text-decoration or border-bottom as links in the world wide web are always with a line underneath and users expect that behaviour.